packages/react/src/metadata.tsx

```diff
@@ -206,27 +206,27 @@ export const useActionMetadata = (actionFunction: ActionFunction<any, any, any,
  */
 export const filterFieldList = (
   fields: FieldMetadata[] | undefined,
-  options?: { include?: string[]; exclude?: string[]; fields?: string[]; select?: Record<string, any> }
+  options?: { include?: string[]; exclude?: string[] }
 ): FieldMetadata[] => {
   if (!fields) {
     return [];
   }
+
   let subset = fields;
-  if (options?.fields) {
```

Comment:
Seems to be the same as `include` but throwing the exception becomes problematic when the action fields metadata is loading